src/ducks/user.duck.js

```diff
@@ -327,7 +328,17 @@ export const fetchCurrentUser = (params = null) => (dispatch, getState, sdk) =>
 
   const parameters = params || {
     include: ['profileImage', 'stripeAccount'],
-    'fields.image': ['variants.square-small', 'variants.square-small2x'],
+    'fields.image': ['variants.avatar', 'variants.avatar2x'],
```

Comment:
Since this is in user.duck.js, does this affect the ProfileSettingsPage too? I thought that you would only add new variants.